app/js/controllers/main.js

```diff
@@ -50,22 +48,34 @@ export default class MainController extends Controller {
       return;
     }
 
-    this.view.customizer.setRootNode(document.documentElement);
+    // Insert the customizer view into the app's document
     document.body.appendChild(this.view.el);
 
-    var padding = this.view.el.querySelector('fxos-customizer').offsetHeight;
-    this._styleEl = document.createElement('style');
-    this._styleEl.textContent = `
-    html, body {
-      overflow: initial !important;
-    }
+    // And display it. This has to be in a setTimeout or the CSS animation
+    // won't work correctly since we just added it above.
+    setTimeout(() => { this.view.show(); });
```

Comment:
I've always found this.f.bind(this) to be repetitive and ugly; kind of an anti-pattern. And arrow functions allow us to avoid it and still be concise. Since both create a new function, I assume that they have similar performance characteristics, and I'd prefer to just use => instead of bind in situations like this. I think it makes the code clearer.